frontend/src/locales/de.json

```
@@ -366,6 +366,7 @@
         "title": "Dein Account wird aktiviert"
       },
       "login": {
+        "acceptTermsOfUseOnOAuthLogin": "Beim Login via einen diese Services akzeptierst du die {termsOfServiceLink}.",
```

```suggestion
        "acceptTermsOfServiceOnOAuthLogin": "Beim Login via einen diese Services akzeptierst du die {termsOfServiceLink}.",
```
TermsOfService to be consistent?

---

[6f049c7](https://github.com/ecamp/ecamp3/pull/3341/commits/6f049c7bda61a8aebb66d5abd723563ae592f3bd)

I had taken this term over it from the Register component. So to be really consistent, I also adjusted it there.